coercions/packagesFromJsonFileCoercion.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+let fs = require('fs'),
+    packages = require('./packagesCoercion');
+
+module.exports = function (jsonFilePath) {
+    const fileContent = JSON.parse(fs.readFileSync(jsonFilePath, "utf8"));
+    const dependencies = attachNameAndVersion(fileContent.dependencies);
+    const devDependencies = attachNameAndVersion(fileContent.devDependencies);
```

Comment:
You should check if the download dev-dependencies flag is true, its false by default.